zhaquirks/tuya/tuya_trv.py

```diff
@@ -696,7 +696,8 @@ async def write_attributes(
                     TuyaPresetMode.Heat: Thermostat.SystemMode.Heat,
                     TuyaPresetMode.Off: Thermostat.SystemMode.Off,
                 }[x],
-                dp_converter=lambda x: {
+                dp_converter=lambda x,
+                _: {  # TODO: add a new dp_converter argument to tuya_dp_multi and remove from this
```

Comment:
To keep this PR small, I plan to add support for this in the builder in a separate PR. For now, it needs the extra argument since this DP has 2 attributes mapped.
If this is not fine, I can do the Builder changes in this PR.

Reply:
The changes were smaller than I anticipated, so updated this one.